components/RPC-Connection.jsx

```diff
@@ -143,6 +143,9 @@ function applyFilter(value, filter, network, setReturnValue) {
 		case "blocksToDays":
 			BlocksToDays(value, setReturnValue);
 			break;
+		case "arrayLength":
+			ArrayLength(value, setReturnValue);
+			break;
```

Comment:
This is from https://github.com/w3f/polkadot-wiki/pull/3605 but was omitted from https://github.com/w3f/polkadot-wiki/pull/4074.